README.md

```diff
@@ -56,6 +64,8 @@ Create a `secretproviderclasses` resource to provide provider-specific parameter
       provider: azure                   # accepted provider options: azure or vault
       parameters:
         usePodIdentity: "false"         # [OPTIONAL for Azure] if not provided, will default to "false"
+        useVMManagedIdentity: "false"   # [OPTIONAL] if not provided, will default to "false"
```

Comment:
Can we add this `# [OPTIONAL available for version > 0.0.4]` for both `useVMManagedIdentity` and `userAssignedIdentityID` so users are aware which version this is available with?